source/configuration_manual/stats/index.rst

```diff
@@ -272,6 +272,15 @@ Metrics can be removed dynamically by running:
    doveadm stats remove <name>
 
 Examples:
+
+.. code-block:: none
+
+   doveadm stats add --description "Description of the metric" --exporter log-exporter --exporter-include "name timestamps" --fields "bytes_in bytes_out" --group-by "cmd_name tagged_reply_state" metric_name "event=imap_command_finished AND name=SELECT"
```

Comment:
Since this is an example, the description would be better as a real description. Like "IMAP SELECT commands". And metric_name -> imap_cmd_select?